ui/app/styles/components/linkable-item.scss

```diff
@@ -0,0 +1,21 @@
+.linkable-item {
+  display: grid;
+  grid-template-columns: minmax(0, 1fr) 50px 35px;
```

Comment:
nice use of css grid!

Reply:
@chelshaw came up with this stellar idea to wrangle some pesky overflowing text